### PR TITLE
Update documentation for the Bring Your Own Container feature

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

**Review by binura-g — APPROVED** [review]
Skimmed over it, looks okay for now. It'll be easier to go through it once deployed.

**Comment by RukshiW** [comment]
Merging this PR as requested by Binura.

Once it is merged, please use the Development environment for Choreo Documentation [1] to view the changes because the changes will not be available in the Production environment until a Sync is done.

The related GitHub issue [2] will be kept open until all the feedback is received and addressed.

[1] https://docs.dv.choreo.dev/
[2] https://github.com/wso2-enterprise/choreo/issues/15320

**Comment by RukshiW** [comment]
Connected PR https://github.com/wso2/docs-choreo-dev/pull/620

File: en/docs/devops-portal.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+- Clicking **DevOps Portal** in the top menu of the Choreo Console
+
+   ![Top menu link](assets/img/devops-portal/top-menu-link.png){.cInlineImage-full}
+
+When you open the DevOps Portal, it displays the deployment insights and the recently created components of the currently selected project as shown below:
+
+![DevOps Portal landing page](assets/img/devops-portal/devops-portal-landing-page.png){.cInlineImage-threeQuarter}
+
+If you hold the pointer over the **Deployment Insights** graph, it displays the number of components you have deployed to each environment up to the date you are pointing. This count applies to the currently selected organization and project.
```
Comment (shaniR) [inline en/docs/devops-portal.md:19]: Up to date?

File: en/docs/devops-portal.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+On this page, you can view the information described in the following subtopics.
+
+### Overview
+
+This tab displays the following information:
+
+![Component overview in the private data plane](assets/img/devops-portal/devops-component-overview-in-private-data-plane.png){.cInlineImage-full}
+
+!!! Tip
+    The internal routing URL is an internal endpoint for the component. Other components in the same organization can use this URL to communicate with this component without using the public internet or an internal gateway.
```
Comment (shaniR) [inline en/docs/devops-portal.md:38]: What is meant by this component?

File: en/docs/devops-portal.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+This tab displays configurations and/or secrets that you have configured for your component.
+
+You can also add new configurations and/or secrets.
+
+![Create a configuration mount](assets/img/devops-portal/create-a-configuration-mount.png){.cInlineImage-full}
+
+### Health Checks
+
+This tab allows you to define health checks for a selected container.
```
Comment (shaniR) [inline en/docs/devops-portal.md:78]: what type of health checks?

File: en/docs/devops-portal.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+This tab displays information related to a selected Docker container such as the URL to the Docker image from which Choreo pulls the component, the image pull policy, resource limits and requests, exposed ports, etc.
+
+If you are on a private plane, you can edit this information. You can also add commands and arguments to the Docker image.
+
+### Configurations and secrets
+
+This tab displays configurations and/or secrets that you have configured for your component.
+
+You can also add new configurations and/or secrets.
```
Comment (shaniR) [inline en/docs/devops-portal.md:72]: shall we say encrypted secrets?

File: en/docs/devops-portal.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### Health Checks
+
+This tab allows you to define health checks for a selected container. 
+
+![Create health check](assets/img/devops-portal/create-a-health-check.png){.cInlineImage-half}
+
+### Scaling
+
+In this tab, you can define the thresholds based on which the component can scale horizontally.
```
Comment (shaniR) [inline en/docs/devops-portal.md:84]: thresholds based on which the component can scale horizontally -> based on the horizontal scaling thresholds.

**Pull request merged**